### PR TITLE
adding files for the user_subjects and user_tasks table.

### PR DESCRIPTION
**Pull request by naid**

### CODE REVIEW

File: app/Models/UserSubject.php [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+use Illuminate\Database\Eloquent\Model;
+
+class UserSubject extends Model
+{
+    public function user()
+    {
+        return $this->belongsTo(User::class, 'user_id');
+    }
+    public function course()
```
Comment (dinhhoanglong91) [inline app/Models/UserSubject.php]: remove it

File: app/Models/UserTask.php [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+class UserTask extends Model
+{
+    public function user()
+    {
+        return $this->belongsTo(User::class, 'user_id');
+    }
+    public function course()
+    {
+        return $this->belongsTo(Course::class, 'course_subjects', 'course_id');
```
Comment (wataridori) [inline app/Models/UserTask.php]: Is this definition right?
The second parameter in `belongsTo` method should be a foreign key, but I think 'course_subjects' is the table name?

File: database/migrations/2016_02_18_034248_create_user_subjects_table.php [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('user_subjects', function (Blueprint $table) {
+            $table->increments('id');
+            $table->integer('user_id')->unsigned()->index();
+            $table->integer('course_id')->unsigned()->index();
+            $table->integer('subject_id')->unsigned()->index();
+            $table->string('status')->nulllable();
```
Comment (wataridori) [inline database/migrations/2016_02_18_034248_create_user_subjects_table.php]: I think you should use number for the `status` field, not string.

File: database/seeds/ActivitiesTableSeeder.php [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+    {
+        $faker = Faker\Factory::create();
+
+        for ($i = 0; $i <= 10; $i++) {
+            $activityData = [
+                'user_id' => 1,
+                'description' => $faker->paragraph(1),
+            ];
+        }
+        Activity::create($activityData);
```
Comment (wataridori) [inline database/seeds/ActivitiesTableSeeder.php]: If you use `Activity::create()` here, you will only create one record. You should move it into the `for` loop.

File: database/migrations/2016_02_18_034248_create_user_subjects_table.php (hunk 3 above)
Comment (wataridori) [inline database/migrations/2016_02_18_034248_create_user_subjects_table.php]: And you should define status const inside the model.
Such as:
```php 
const STATUS_START = 1;
const STATUS_COMPLETE = 2;
```

File: database/seeds/UserTasksTableSeeder.php [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+     */
+    public function run()
+    {
+        $setData = [
+            [
+                'task_id' => 1,
+                'user_id' => 1,
+                'course_id' => 1,
+                'subject_id' => 1,
+                'status' => 'start',
```
Comment (wataridori) [inline database/seeds/UserTasksTableSeeder.php]: Do not hard code a string like this. You'd better define and use const.
Please refer https://github.com/framgia/tms_47/pull/5/files#r53290223

File: database/seeds/UserTasksTableSeeder.php [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+            [
+                'task_id' => 1,
+                'user_id' => 3,
+                'course_id' => 2,
+                'subject_id' => 2,
+                'status' => 'start',
+                'end_date' => Carbon::now(),
+            ],
+        ];
+        Task::insert($setData);
```
Comment (wataridori) [inline database/seeds/UserTasksTableSeeder.php]: Function `insert` will not generate `created_at` and `updated_at` value. You should assign the value yourself.

**Pull request closed without merging**